src/pages/our-work.js

```diff
@@ -43,9 +43,11 @@ const OurWork = ({ data }) => {
   const engineeringCaseStudies = formatCaseStudies(
     allContentfulTemplatedCaseStudy
   )
+
   const designCaseStudies = formatCaseStudies(
     allContentfulNonTemplatedCaseStudy
   )
+
```

Comment:
You'll want to add `allContentfulNonTemplatedCaseStudyV2` to this as well

Reply:
👍

Reply:
It's actually already there.

Reply:
It is in your GraphQL query yes but the data it returns is not being used in the OurWork component so you're not seeing the doctor link project in the case study list on `/our-work`

```
  const {
    allContentfulTemplatedCaseStudy,
    allContentfulNonTemplatedCaseStudy,
    allContentfulNonTemplatedCaseStudyV2 <--- this bit here is missing in L37
  } = data
```

Reply:
done